codecov.yml

```diff
@@ -31,6 +34,6 @@ flags:
 
   front-end:
     paths:
-      - enterprise/frontend
-      - frontend
+      - enterprise/frontend/src
+      - frontend/src
```

Comment:
I want to add a note that this probably won't change anything afaik.
<img width="684" alt="image" src="https://user-images.githubusercontent.com/1937582/221581511-e7ab3839-de7c-4d89-a35d-b2b84fadd3d1.png">

As both `enterprise/frontend` and `frontend` are already only containing `src` as their only single child.

Reply:
Yeah, it doesn't hurt to be explicit.

Good to know this, nevertheless. Thanks @WiNloSt